content/en-us/tutorials/fundamentals/coding-5/intro-to-dictionaries.md

```diff
@@ -90,15 +90,15 @@ There are two ways to access dictionary values:
 `tableName["keyName"]` -- Note the quotations
 `tableName.keyName`
 
-```lua
-local enemy = {
-   Name = "Spike",
-   Health  = 1000,
-}
+   ```lua
+   local enemy = {
+      Name = "Spike",
```

Comment:
Spacing doesn't seem right here, review all these spacing changes on this please